File: trunk/main/test/com/seovic/coherence/identity/CoherenceUuidGeneratorTests.java
package com.seovic.coherence.identity;


import com.seovic.identity.IdentityGeneratorClient;

import static org.junit.Assert.*;

import org.junit.Test;


/**
 * Tests for {@link CoherenceUuidGenerator}.
 *
 * @author Aleksandar Seovic  2008.11.24
 */
public class CoherenceUuidGeneratorTests
    {
    @Test
    public void testIdGeneration()
        {
        IdentityGeneratorClient igc = new IdentityGeneratorClient(
                new CoherenceUuidGenerator());

        assertEquals(100, igc.generateIdentities(1, 100).size());
        assertEquals(100, igc.generateIdentities(5, 20).size());
        assertEquals(100, igc.generateIdentities(10, 10).size());
        }
    }

File: trunk/main.net/test/Coherence.Tools.Tests/Coherence/Identity/CoherenceUuidGeneratorTests.cs
using NUnit.Framework;
using Seovic.Identity;

namespace Seovic.Coherence.Identity
{
    [TestFixture]
    public class CoherenceUuidGeneratorTests
    {
        [Test]
        public void TestIdGeneration()
        {
            IdentityGeneratorClient<Tangosol.Util.UUID> igc = new IdentityGeneratorClient<Tangosol.Util.UUID>(
                new CoherenceUuidGenerator());

            Assert.AreEqual(100, igc.GenerateIdentities(1, 100).Count);
            Assert.AreEqual(100, igc.GenerateIdentities(5, 20).Count);
            Assert.AreEqual(100, igc.GenerateIdentities(10, 10).Count);
        }
    }
}